modules/reviewing-cluster-status-from-the-openshift-cluster-manager.adoc

```diff
@@ -9,13 +9,11 @@
 From the {product-title} web console, you can review detailed information about the status of your cluster on {cluster-manager}.
 
 .Prerequisites
-
+* You are logged in to the {cluster-manager-url}.
 * You have access to the cluster as a user with the `cluster-admin` role.
 
 .Procedure
-
-. In the *Administrator* perspective, navigate to *Home* -> *Overview* -> *Details* -> *Cluster ID* -> *{cluster-manager}* to open your cluster's *Overview* tab in the {cluster-manager} web console.
-
+. Go to the *Clusters* list in the {cluster-manager-url} and locate your {product-title} cluster.
 . From the *Overview* tab on {cluster-manager-url}, review the following information about your cluster:
```

Comment:
This could use clarifying, since there's now an Overview item on the left nav as well, but we are talking about the Overview tab for the cluster. (ie. you click the cluster, then you view the Overview for that cluster)